# HG changeset patch
# User Rik <rik@octave.org>
# Date 1485389078 28800
#      Wed Jan 25 16:04:38 2017 -0800
# Branch stable
# Node ID b19287c4e74a4dba880571d135ed575a11b665d0
# Parent  e9a0469dedd9dfd0ee1016b9ee8fabb81fd50b4f
doc: Reword documentation about subplots in 15.2.4 (bug #50148).

* plot.txi: Reword documentation about subplots in 15.2.4.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -520,21 +520,23 @@ For example,
 subplot (2, 1, 1)
 fplot (@@sin, [-10, 10]);
 subplot (2, 1, 2)
 fplot (@@cos, [-10, 10]);
 @end group
 @end example
 
 @noindent
-creates a figure with two separate axes, one displaying a sine wave and
-the other a cosine wave.  The first call to subplot divides the figure
-into two plotting areas (two rows and one column) and makes the first plot
-area active.  The grid of plot areas created by @code{subplot} is
-numbered in column-major order (top to bottom, left to right).
+creates a figure with two separate axes, one displaying a sine wave and the
+other a cosine wave.  The first call to subplot divides the figure into two
+plotting areas (two rows and one column) and makes the first plot area active.
+The grid of plot areas created by @code{subplot} is numbered in row-major order
+(left to right, top to bottom).  After plotting a sine wave, the next call to
+subplot activates the second subplot area, but does not re-partition the
+figure.
 
 @DOCSTRING(subplot)
 
 @node Multiple Plot Windows
 @subsection Multiple Plot Windows
 @cindex plotting, multiple plot windows
 
 You can open multiple plot windows using the @code{figure} function.
